src/api/public/apidocs-new/paths/person_login.yaml

```diff
@@ -1,3 +1,36 @@
+get:
+  summary: Get details about a person
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/login.yaml'
+  responses:
+    '200':
+      description: >
+        OK. The request has succeeded.
+
+        XML Schema used for body validation: [person.xsd](../schema/person.xsd)
```

Comment:
The link to _person.xsd_ is broken. I couldn't find that file in the project.

Reply:
Fixed